integration/emr/alluxio-emr.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+#Create user and download release
+sudo groupadd alluxio -g 600
+sudo useradd alluxio -u 600 -g 600
+if [ -z $1]
+then
+  wget http://downloads.alluxio.io/downloads/files/2.0.0-preview/alluxio-2.0.0-preview-bin.tar.gz
```

Comment:
any way to modify the user-agent? you can leave a TODO here too if it takes more time to figure out

Reply:
Left a TODO. Maybe a simple request header will do.